client/components/Events/EventDateTime.tsx

```diff
@@ -23,6 +24,7 @@ export class EventDateTime extends React.PureComponent<IProps> {
         const end = eventUtils.getEndDate(item);
         const isAllDay = eventUtils.isEventAllDay(start, end);
         const multiDay = !eventUtils.isEventSameDay(start, end);
+        const showEventStartDate = eventUtils.showEventStartDate(start, this.props.planningItem?.date);
```

Comment:
So if `planningItem.date` is `null`, this will continue working?

Reply:
yep